cypress/e2e/gui/messenger.cy.js

```diff
@@ -0,0 +1,43 @@
+describe('Messenger', () => {
+
+  beforeEach(() => {
+    cy.startSession()
+  })
+
+  it('Open and close the messenger', () => {
+    cy.get('button[class*="Messenger_openCloseButton"]')
+      .click();
+    cy.contains('h2', 'How can we help you?')
+      .should('be.visible')
+    cy.get('button[class*="Messenger_openCloseButton"]')
+      .click()
+  })
+
+  it.only('Ensures that all messenger fields are required and that the first one is focused', () => {
```

Comment:
```suggestion
  it('Ensures that all messenger fields are required and that the first one is focused', () => {
```

Cuidado para não esquecer testes com `.only`!

Reply:
Vou revisar todos os cenários antes subir o PR mestre